.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -28,87 +28,89 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 public class MavenWrapperDownloader {
```

Comment:
revert changes (or gitignore), if it's in repo already then I would recommend revert changes as it is in master branch